SMLHelper/Assets/SubnauticaModSprite.cs

```diff
@@ -19,6 +19,8 @@ internal static void Add(SpriteManager.Group group, string name, Atlas.Sprite sp
             if (!ModSprites.ContainsKey(group))
                 ModSprites.Add(group, new Dictionary<string, Atlas.Sprite>(StringComparer.InvariantCultureIgnoreCase));
 
+            if(ModSprites[group].ContainsKey(name))
+                Logger.Debug($"ModSprite already registered for {group}/{name}.  Old sprite will be overwritten.");
```

Comment:
In the case of a conflict, shouldn't both entries be thrown out, similarly to what happens with other data types?

Reply:
That would 1. alter the effect that was already present. and 2. cause whatever mods added those sprites to fail to find them when they looked causing the ? icon instead.

Reply:
this way just adds logging to the debug level to show WHEN one mod is overwriting another mods sprite.

Reply:
@AlexejheroYTB  We no longer discard both entries when in conflict (to avoid worse problems).  
We keep the latest entry and log when an overwrite took place..